client/etcd/client.go

```diff
@@ -98,7 +98,15 @@ func (c *csclient) Services() (services.Services, error) {
 
 func (c *csclient) KV() (kv.Store, error) {
 	c.kvOnce.Do(func() {
```

Comment:
Might be better to move the logic inside `KV` and `Txn` into an unexported internal function?